### PR TITLE
Leaks2

### PR DESCRIPTION
**Pull request by tbzatek**
Another bunch of small fixes. There are still a couple of left, to be added (given that they aren't udisks issues in fact...).

### CODE REVIEW

**Comment by tbzatek** [comment]
Okay, I think I'm done with memory leaks hunting. There are couple of minor ones left in udisks, nothing in libblockdev so far.

File: src/plugins/mdraid.c [hunk 1]
```diff
@@ -357,8 +357,7 @@ static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean
         }
 
         if (bs_error) {
-            g_set_error (error, BD_MD_ERROR, BD_MD_ERROR_PARSE,
-                         "Failed to parse chunk size from mdexamine data: %s", bs_error->msg);
+            g_warning ("get_examine_data_from_table(): Failed to parse chunk size from mdexamine data: %s", bs_error->msg);
```
Comment (vpodzime) [inline src/plugins/mdraid.c]: Why this change?

File: src/plugins/mdraid.c [hunk 2]
```diff
@@ -275,7 +275,7 @@ static GHashTable* parse_mdadm_vars (const gchar *str, const gchar *item_sep, co
     return table;
 }
 
-static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean free_table, GError **error) {
+static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean free_table, G_GNUC_UNUSED GError **error) {
```
Comment (vpodzime) [inline src/plugins/mdraid.c]: This should be squashed with the previous commit. And I believe the `error` should actually be used.

File: src/plugins/mdraid.c (hunk 1 above)
Comment (tbzatek) [reply src/plugins/mdraid.c]: Ah, that. We've agreed with @vojtechtrefny that this shouldn't fail the call - it's a single attribute that is not that important when missing. This change also mimics the previous behaviour where data have been returned successfully and the error got simply lost.

File: src/plugins/mdraid.c (hunk 2 above)
Comment (tbzatek) [reply src/plugins/mdraid.c]: Squashed. We could perhaps make use of `error` by defining couple of keys as mandatory. Just a thought.

File: src/plugins/mdraid.c (hunk 1 above)
Comment (vpodzime) [reply src/plugins/mdraid.c]: OK

File: src/plugins/mdraid.c (hunk 2 above)
Comment (vpodzime) [reply src/plugins/mdraid.c]: We don't have to do that now.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.
